include/rmm/mr/device_memory_resource.hpp

```diff
@@ -23,6 +23,15 @@ using cudaStream_t = struct CUstream_st*;
 
 namespace rmm {
 namespace mr {
+
+/**
+ * Align up to a power of 2, align_bytes is expected to be a nonzero power of 2
+ */
+inline std::size_t align_up(std::size_t v, std::size_t align_bytes) noexcept
```

Comment:
Suggest moving this up to an anonymous namespace.

Reply:
Should be `detail`. Don't want an anonymous namespace in a header file.

Reply:
Any suggestion for the fiule this should go in ? There doesn't appear to be a detail namespace, and include/rmm/detail only appears to contain memory_manager.hpp, which includes device_memory_resource.hpp

Reply:
Jake just means keep it in this file, but add a `namespace detail` scope around this function.

Reply:
At least I assume that's what he means. :)

Reply:
Sounds reasonable ~ updated the file.